root/doc/DocPage.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import Layout from '../layout';
+
+import DocSearchBox from './components/DocSearchBox';
+
+type DocPageT = {|
+  +content: string,
+  +hierarchy: $ReadOnlyArray<string>,
+  +title: string,
+  +version: number,
+|};
+
+type Props = {|
+  +id: string,
+  +page: DocPageT,
+  +useGoogleCustomSearch: boolean,
+  +wikiServer: string,
+|};
+
+const DocPage = ({
+  id,
+  page,
+  useGoogleCustomSearch,
+  wikiServer,
+}: Props) => {
+  let doc = '';
+  let lastDoc = '';
+  const wikiPageUrl = `//${wikiServer}/${id}`;
```

Comment:
I was going to use encodeURIComponent() here, but it messes up with the slashes, so hopefully we don't need to.